projects/auth0-angular/src/lib/auth.interceptor.ts

```diff
@@ -72,34 +88,31 @@ export class AuthHttpInterceptor implements HttpInterceptor {
     route: ApiRouteDefinition,
     request: HttpRequest<any>
   ): boolean {
-    const testPrimitive = (value: string | RegExp) => {
+    const testPrimitive = (value: string) => {
       if (!value) {
         return false;
```

Comment:
should we do an extra .trim() below here to get rid of pre/pos whitespace and \r?

Reply:
Sure, I put that check in